src/main/java/org/owasp/esapi/reference/DefaultValidator.java

```diff
@@ -54,17 +54,46 @@
 import org.owasp.esapi.reference.validation.StringValidationRule;
 
 /**
- * Reference implementation of the Validator interface. This implementation
- * relies on the ESAPI Encoder, Java Pattern (regex), Date,
+ * Reference implementation of the {@code Validator} interface. This implementation
+ * relies on the ESAPI {@code Encoder}, {@link java.util.regex.Pattern},
+ * {@link java.util.Date},
  * and several other classes to provide basic validation functions. This library
- * has a heavy emphasis on whitelist validation and canonicalization.
+ * has a heavy emphasis on allow-list validation and canonicalization.
+ * <p>
+ * <b>A Note about Canonicalization</b>:
+ * The behaviors of objects of this class are largely driven by how the
+ * associated {@code Encoder} is created and passed to one of this
+ * class' constructors. Specifically, what {@link org.owasp.esapi.codecs.Codec}
+ * types are what referenced by the {@link org.owasp.esapi.Encoder} instance
```

Comment:
I think the "what" before "referenced" needs removing

Reply:
Done